### ISSUE TITLE
Clean Localstorage by Containers

### ISSUE THREAD

**Issue by ke-d** [mrdokenny/Cookie-AutoDelete#347]
This is a follow up to #44. I consider this to be a main blocker for enabling Localstorage support by default in CAD.

Relevant Bug:
[1353726](https://bugzilla.mozilla.org/show_bug.cgi?id=1353726)

**Comment by kennethtran93** [comment]
This is still an ongoing issue outside of this extension, and thus will close it to keep open issues close to those that can be worked upon.

I will update the wording within the settings, however.

**Comment by SimonPilkington** [comment]
This should now be doable in Firefox according to https://bugzilla.mozilla.org/show_bug.cgi?id=1353726#c13
Would you consider revisiting?

**Comment by kennethtran93** [comment]
Thanks for the update.  I'll look into it alongside the patch for strict mode / cross-site cookies.

**Comment by kennethtran93** [comment]
Guessing the question now is how to implement this...

**Comment by SpaceClicker** [comment]
Has it been confirmed that this is now possible to implement in Firefox?

This is one of the major pain points for this extension (that it cannot clear LocalStorage or IndexedDB on a per-container basis) and it would be great to have this ability in this extension.

**Comment by SimonPilkington** [comment]
> Has it been confirmed that this is now possible to implement in Firefox?

Yes, been in Release since 92: https://bugzilla.mozilla.org/show_bug.cgi?id=1599979

**Comment by kennethtran93** [comment]
Somebody needs to update the mdn docs too...I'm not one to keep checking bugzilla if it was implemented, unless it is still only for Nightly, in which case it would still be on the lower end of the priority.
![image](https://user-images.githubusercontent.com/6724477/193434964-fbcd457e-6a2d-4722-85d8-2577c19e8372.png)
